### PR TITLE
first pass ad deploy strategy overview docu

### PR DESCRIPTION
**Pull request by ghedamat**
/cc @achambers @lukemelia 

just a first pass

also have this on a gdoc that I can share w/you guys for easier collaboration

### CODE REVIEW

**Comment by ghedamat** [comment]
mostly @achambers work :) will add links shortly to plugins shortly and then it could be ready

**Comment by ghedamat** [comment]
ok @achambers and @lukemelia I think we're in a good place here merge if happy

File: docs/v0.5.x/deployment-strategies-overview.md [hunk 1]
```diff
@@ -1,3 +1,61 @@
...
+
+### Source maps
+Do you want to upload source maps to a bug reporting service or something other hosted service to make it easier to debug minified JS? This can often be handy when using services like BugSnag, Sentry and Raygun.
+
+### Using a manifest file
+Manifest files are helpful when uploading assets. Often some assets remain unchanged in which case there is no benefit in uploading them again. A manifest file is used to keep track of which files have been uploaded so that they don't need to be re-uploaded next time you deploy. We have a plugin for this called [ember-cli-deploy-manifest](https://github.com/ember-cli-deploy/ember-cli-deploy-manifest).
+
+### Notifying team members of deployments
+Once a deploy has finished successfully (or maybe failed as the case may be) you may find it handy to notify your team. Currently we have the [ember-cli-deploy-slack](https://github.com/dschmidt/ember-cli-deploy-sentry) plugin to send notifications to Slack. However if you use a different platform or prefer to send emails instead, try writing your own plugin.
```
Comment (lukemelia) [inline docs/v0.5.x/deployment-strategies-overview.md]: URL is wrong

**Comment by ghedamat** [comment]
@lukemelia should be fixed

**Pull request merged**